packages/web/package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "@sabia/web",
   "scripts": {
-    "dev": "APP_ENV=development node server.js",
-    "build": "next build",
-    "start:ci": "APP_ENV=testing NODE_ENV=production npm run build && APP_ENV=testing NODE_ENV=production npm run start",
-    "start": "NODE_ENV=production node server.js",
+    "dev": "NODE_ICU_DATA=node_modules/full-icu APP_ENV=development node server.js",
+    "build": "NODE_ICU_DATA=node_modules/full-icu next build",
+    "start:ci": "APP_ENV=testing NODE_ENV=production npm run build && APP_ENV=testing npm run start",
+    "start": "NODE_ICU_DATA=node_modules/full-icu NODE_ENV=production node server.js",
```

Tive que especificar o `icu` também nos scripts de dev/prod pois como a primeira renderização é server side, o node por padrão só suporta o locale `en-us`

---

Awesome "traquinagem"! I think you can add a "base script" with ICU data to get better readability.
```suggestion
    "icu": "NODE_ICU_DATA=node_modules/full-icu"
    "dev": "npm run icu -- APP_ENV=development node server.js",
    "build": "npm run icu -- next build",
    "start:ci": "APP_ENV=testing NODE_ENV=production npm run build && APP_ENV=testing npm run start",
    "start": "npm run icu -- NODE_ENV=production node server.js",
```
You will need to test if you really need the `--` here ☝️

---

@luizeboli, have you tried to deploy this to our staging environment? I guess it won't break, but we have to make sure...

---

@lcnogueira no, not in stage. But things break if we try to run test in local environment, that's the need of these blocks

---

@mateus4k Sadly none of these options work, neither `npm run icu -- ...` nor `npm run icu && ..`. We'll have to discuss a better way to improve these scripts readability.